README.md

```diff
@@ -76,6 +76,17 @@ react-native link react-native-qrcode-scanner
 react-native link react-native-permissions
```

Comment:
can you move this link step and the install step in a few lines above (line 71 to be more specific) to the `react-native-permissions` section as you've created? it sounds a better place now to have these both instructions steps

Reply:
Done, let me know if you want any other changes